### PR TITLE
Err comp config pairs code

### LINKED ISSUE #10762: [Server] Meshery Server - ErrCompConfigPairsCode-1073:  Missing probable cause and suggested remediation
### Current Behavior
Screenshot 2024-04-23 at 10 33 30 AM

### Expected Behavior
Description, cause and remediation are needed.

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v0.7.46
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #10762 

Update documentation and error codes in Go by including missing long descriptions, probable causes, and suggested remedy.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 04e8b86728a79aa6bdf4066afc0e02dcaef1da9e at: https://663b26842cc488d107b159c0--meshery-docs-preview.netlify.app

File: docs/_data/errorref/meshery-server_errors_export.json [hunk 1]
```diff
@@ -655,10 +655,10 @@
         "name": "ErrCompConfigPairsCode",
         "code": "1073",
         "severity": "Alert",
-        "long_description": "",
-        "short_description": "unable to Create Comp Config.",
-        "probable_cause": "",
-        "suggested_remediation": ""
+        "long_description": "An error occurred while trying to create Component Configuration.",
+        "short_description": "Unable to Create Component Configuration.",
+        "probable_cause": "Missing or invalid user token, invalid Kubernetes handler or serviceActionProvider error.",
```
Comment (leecalcote) [inline docs/_data/errorref/meshery-server_errors_export.json:660]: Refactor to make this user-facing. To a user, what is a Kubernetes handler or serviceActionProvider? 

Users know about Meshery Connections and can be offered a link to troubleshooting or understanding connections and Meshery's interactions with Kubernetes (in Meshery Docs), but they will not understand references to function names or golang routines.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 06c4a8c0e87309c2da28576b27526d78cd6a320e at: https://664391e13eb358009ebe2e20--meshery-docs-preview.netlify.app

**Comment by Jougan-0** [comment]
should MeshMap Screenshot Service / MeshMapScreenshot (pull_request_target) 
this fail?

**Comment by l5io** [comment]
🚀 Preview for commit b9a0b2971861b4653b4a9b07ba2333c0f7934867 at: https://664e2de3ab580a0417858800--meshery-docs-preview.netlify.app

File: server/handlers/error.go [hunk 2]
```diff
@@ -207,9 +207,8 @@ func ErrExecutionPlan(err error) error {
 }
 
 func ErrCompConfigPairs(err error) error {
-	return errors.New(ErrCompConfigPairsCode, errors.Alert, []string{"unable to Create Comp Config.", err.Error()}, []string{err.Error()}, []string{}, []string{})
+	return errors.New(ErrCompConfigPairsCode, errors.Alert, []string{"Unable to Create Component Configuration.", err.Error()}, []string{("An error occurred while trying to create Component Configuration."), err.Error()}, []string{"Missing or invalid user token, or an issue with Meshery's interactions with Kubernetes."}, []string{"For understanding and troubleshooting, refer to https://docs.meshery.io/guides/troubleshooting and https://docs.meshery.io/installation/kubernetes"})
```
Comment (MUzairS15) [inline server/handlers/error.go:210]: Add name of the component for which it failed, (its kind and name).
> "Missing or invalid user token, or an issue with Meshery's interactions with Kubernetes." 
only single cause?
1. No registered model/comp  found inside registry
2. failed validation

Will you verify what error you get if we try to deploy a CR inside a cluster which doesn't have a CRD deployed.

What error you get from meshery vs when you do same from kubectl

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Review by leecalcote — APPROVED** [review]

**Comment by Jougan-0** [comment]
Closing this as @singh1203 is working on similar issue and is updating the error.

**Pull request closed without merging**